### PR TITLE
Modern libvirt

### PR DESCRIPTION
**Pull request by mikalstill**

### CODE REVIEW

File: ansible/files/libvirt.tmpl [hunk 1]
```diff
@@ -4,56 +4,53 @@
   <memory unit='KiB'>{{memory}}</memory>
   <currentMemory unit='KiB'>{{memory}}</currentMemory>
   <vcpu placement='static'>{{vcpus}}</vcpu>
+
+  <sysinfo type='smbios'>
+    <system>
+      <entry name='manufacturer'>Shaken Fist</entry>
+      <entry name='product'>shakenfist</entry>
+      <entry name='version'>0.2</entry>
```
Comment (mikalstill) [inline ansible/files/libvirt.tmpl:12]: I need to do a better job of this bit.

File: ansible/terraform/aws-single-node/setup.tf [hunk 2]
```diff
@@ -25,7 +25,7 @@ data "aws_ami" "ubuntu" {
 
   filter {
     name   = "name"
-    values = ["ubuntu/images/hvm-ssd/ubuntu-bionic-18.04-amd64-server-*"]
+    values = ["ubuntu/images/hvm-ssd/ubuntu-*-20.04-amd64-server-*"]
```
Comment (mandoonandy) [inline ansible/terraform/aws-single-node/setup.tf:28]: Are we moving all support to 20.04?

**Review by mandoonandy — COMMENTED** [review]
Just asking since we have been so busy. These changes look like testing attempts, esp CPU downgrade and timer change. The USB removal might one day be problematic if an exploit wants to use some USB vulnerability.

File: ansible/files/libvirt.tmpl [hunk 3]
```diff
@@ -4,56 +4,53 @@
...
+  <on_crash>destroy</on_crash>
+
   <devices>
     <emulator>/usr/bin/kvm</emulator>
 
     {% for disk in disks %}
     <disk type='file' device='{{disk.present_as}}'>
       <driver name='qemu' type='{{disk.type}}'/>
       <source file='{{disk.path}}'/>
+      {{disk.backing}}
```
Comment (mikalstill) [inline ansible/files/libvirt.tmpl:46]: We needed this one urgently for a more modern libvirt, so I've hotfixed it in.

**Comment by mandoonandy** [comment]
Hmm. This has hang around for a bit. Should I have merged this yet?
Of course, it now has conflicts. Do you want to fix and I'll merge it.

**Comment by mikalstill** [comment]
Please don't merge this. I don't think we need it any more, the good bits are in other PRs.

**Pull request closed without merging**